rockcraft.yaml

```diff
@@ -33,8 +37,36 @@ parts:
 
   certificates:
     plugin: nil
-    stage-packages:
+    build-packages:
       - ca-certificates
+    override-build: |
+      mkdir -p $CRAFT_PART_INSTALL/etc/ssl/certs
+      mkdir -p $CRAFT_PART_INSTALL/usr/share/ca-certificates/mozilla/
+      mkdir -p $CRAFT_PART_INSTALL/usr/sbin
+      mkdir -p $CRAFT_PART_INSTALL/tmp
+      
+      touch $CRAFT_PART_INSTALL/etc/ssl/certs/ca-certificates.crt
+      touch $CRAFT_PART_INSTALL/etc/ca-certificates.conf
+      for cert in /usr/share/ca-certificates/mozilla/* ; do
+        echo "mozilla/$(basename $cert)" >> $CRAFT_PART_INSTALL/etc/ca-certificates.conf
+        cat "$cert" >> $CRAFT_PART_INSTALL/etc/ssl/certs/ca-certificates.crt
```

Comment:
question: Why are we creating this file? I don't see it at https://sources.debian.org/src/ca-certificates/20230311/debian/ca-certificates.postinst/

Reply:
/etc/ssl/certs/ca-certificates.crt holds all the certs in ca-certificates. This file is supposed to be created by the ca-certificates package's postint script like ca-certificates.conf. The update-ca-certificate script also updates this file as well.